tests/test_result.py

```diff
@@ -344,3 +344,12 @@ def test_posterior_samples(result, random_state):
     empirical_posterior = counts / n_samples
     np.testing.assert_allclose(empirical_posterior, posterior, atol=1e-2)
 
```

Comment:
two spaces, please

Reply:
fixed in 0d2a70e4cb63f5f9b2cc3107edbb0dfde1d8e9a1